packages/venia-concept/src/ServiceWorker/Utilities/htmlHandler.js

```diff
@@ -142,17 +143,75 @@ const cloneRequestWithDiffURL = (request, url) =>
           })
         : request;
 
+/**
+ * Function validates if the script files mentioned in the cached
+ * HTML file are cached. If not, it returns false.
+ *
+ * @returns Promise<Boolean>
+ */
+const areAllHTMLLinksCached = async () => {
```

Comment:
Here we are only validating `script` files. We can also validate for `links` and `CSS` files as well. I have left it with `script` files because their absence will crash the app, but the absence of links or CSS files will not. I will be happy to change it if needed.